lib/libzrepl/data_conn.c

```diff
@@ -891,7 +887,7 @@ uzfs_zvol_rebuild_scanner(void *arg)
 			name = kmem_alloc(hdr.len, KM_SLEEP);
 			rc = uzfs_zvol_socket_read(fd, name, hdr.len);
 			if (rc != 0) {
-				LOG_DEBUG("Error:%d reading volname\n", errno);
+				LOG_ERR("Error reading zvol name");
```

Comment:
do we need errno  too ?

Reply:
the errno is already logged in uzfs_zvol_socket_read. Moreover the connection might have been just closed which means there is no errno error.